crates/supply/src/benchmarking.rs

```diff
@@ -0,0 +1,26 @@
+//! Loans pallet benchmarking.
+#![allow(unused_imports)]
+#![cfg(feature = "runtime-benchmarks")]
```

Comment:
Not needed I think

Reply:
Removed the `runtime-benchmarks` conditional compilation but left `#![allow(unused_imports)]` because the `use crate::Pallet as Supply` import fails without it. I tried conditionally compiling it with the `runtime-benchmarks` features but the import still fails when I run the tests.